src/Network/Network/ChangeLog.md

```diff
@@ -23,6 +23,7 @@
     - Added `CustomV2` to the validation set of `PolicyType`
     - Added `TLSv1_3` to the validation set of `MinProtocolVersion`
     - Removed validation for null or empty cipher suites list since there can be empty cipher suites list for min protocol version of tls1.3
+* [Breaking Change] Changed default value of `-PrivateEndpointNetworkPoliciesFlag` and `-PrivateLinkServiceNetworkPoliciesFlag` to `Disabled` in `New-AzVirtualNetworkSubnetConfig` and `Add-AzVirtualNetworkSubnetConfig`
```

Comment:
PrivateEndpointNetworkPoliciesFlag -> what is trailing flag? Its PrivateEndpointNetworkPolicies.

Reply:
PrivateEndpointNetworkPoliciesFlag is parameter name. It should be mentioned in changelog.